src/django/api/migrations/0153_associate_sectors_with_groups.py

```diff
@@ -0,0 +1,241 @@
+            "Repair",
+            "Technical Services",
+        ],
+        "Transportation": [
+            "Air Transportation",
+            "Equipment",
+            "Ground Passenger Transportation",
+            "Maritime Transportation",
+            "Pipeline Transportation",
+            "Rail Transportation",
+            "Transportation Equipment",
+            "Trucking",
+        ],
+        "Aerospace": ["Aerospace", "Air Transportation"],
+        "Forestry": [
+            "Forestry",
+            "Furniture",
+            "Home Accessories",
+            "Home Furnishings",
+            "Logging",
+            "Manufacturing",
+            "Material Production",
+            "Paper Products",
+            "Wood Products",
+        ],
+        "Press": ["Books", "Paper Products", "Printing"],
+        "Health": [
+            "Health",
+            "Healthcare",
+            "Hospitals",
+            "Medical Equipment & Services",
+            "Nursing",
+            "Pharmaceuticals",
+        ],
+        "Entertainment": [
+            "Arts",
+            "Arts & Entertainment",
+            "Entertainment",
+            "Recreation",
+        ],
+        "Accommodation": ["Accommodation", "Renting"],
+        "Governmental": ["Civics", "Government Registry"],
+        "Finance": ["Banking", "Finance", "Financial Services"],
+        "General Merchandise": [
+            "Commodities",
+            "Consumer Products",
+            "Garden Tools",
+            "General Merchandise",
+            "Hobby",
+            "Merchant Wholesalers",
+            "Multi-Category",
+            "Musical Instruments",
+            "Nondurable Goods",
+            "Plastics",
+            "Rubber Products",
+            "Supplies Dealers",
+            "Toys",
+            "Waste Management",
+            "Wholesale Trade",
+        ],
+        "Storage": ["Storage", "Warehousing"],
+        "Sporting Goods": [
+            "Manufacturing",
+            "Recreation",
+            "Sporting Goods",
+            "Sports Equipment",
+        ],
+        "Education, Research, Services": [
+            "Archives",
+            "Educational Services",
+            "Environmental, Social and Corporate Governance",
+            "International Affairs",
+            "Professional Services",
+            "Research",
+            "Technical and Scientific Activities",
+        ],
+        "Other": [
+            "Allied Products",
+            "Hunting",
+            "Tobacco Products",
+            "Unspecified",
+        ],
+    }
+
+    sectors = Sector.objects.all()
+    for sector in sectors:
+        for group_name, sector_names in sector_groups.items():
+            if sector.name in sector_names:
+                group_instance = SectorGroup.objects.get(name=group_name)
+                sector.groups.add(group_instance)
+
+
+def revert_associate_sectors_with_groups(apps, schema_editor):
+    for sector in Sector.objects.all():
```

Comment:
Similar as above, if we remove the field from model/table the data will be cleared automatically

Reply:
Replaced it to `migrations.RunPython.noop`.